### DISCUSSION TITLE
when the formula for PHP 8 will be released?

### DISCUSSION THREAD

**Discussion by thegr8awais** [Homebrew/discussions#176] [Everyday usage] [answered]
when the formula for PHP 8 will be released?
PHP 8 is released [PHP: PHP 8.0.0 Release Announcement](https://www.php.net/releases/8.0/en.php)

**Comment by jonchang** [comment]
You can search Homebrew/homebrew-core to find out: https://github.com/Homebrew/homebrew-core/pulls?q=is%3Apr+is%3Aopen+php

**Comment by hyuraku** [comment, marked as answer]
PHP 8 is here https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/php.rb
